slamd/discovery/processing/strategies/csv_strategy.py

```diff
@@ -27,10 +32,17 @@ def create_dataset(cls, file_data):
             raise ValueNotSupportedException('You cannot use the name temporary for your dataset!')
 
         try:
-            dataset = Dataset(name=file_name, dataframe=read_csv(file_data, delimiter=delimiter, decimal=decimal))
+            dataset = Dataset(
+                name=file_name,
+                dataframe=read_csv(file_data, delimiter=delimiter, decimal=decimal, on_bad_lines='error')
+            )
         except:
             raise ValueNotSupportedException('The dataset you submitted could not be read.')
 
+        for col in dataset.dataframe.columns:
+            # errors='ignore' => If non-numeric columns can not be converted, they are returned without conversion
```

Comment:
can this lead to a problem in our experiment, e.g. when we ignore a conversion error (in case of an object type column) we might have problems when trying to predict?